lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -931,6 +913,9 @@ export class UserAgentApplication {
 
       // open the window
       const popupWindow = window.open(urlNavigate, title, "width=" + popUpWidth + ", height=" + popUpHeight + ", top=" + top + ", left=" + left);
+      if (!popupWindow || popupWindow == null) {
```

Comment:
2 things 
1.) if popup window is null !popupWindow will be true, the second predicate is unnecessary
2.) (Almost) Always use === strict equals in javascript